src/js/components/dashboard/StockRequestDashboard.jsx

```diff
@@ -31,6 +33,24 @@ const COLUMNS = [
     headerClassName: 'text-left font-weight-bold px-4 py-3',
     className: 'px-4 py-2',
     maxWidth: 200,
+    Cell: (row) => (
+      <span className="d-flex align-items-center">
+        <span className="d-inline-block text-overflow-ellipsis">
+          {row.original.displayStatus}
+        </span>
+        {row.original.recentComment && (
+        <Tooltip
```

Comment:
nitpicky: indent

Reply:
it's weird that ESLint did not throw an error on it, thanks for noticing